src/lib/styles/fonts/plentysans.scss

```diff
@@ -1,31 +0,0 @@
-/**
```

Comment:
Wofür war diese Datei da?

Reply:
die plentysans.scss wurde in der fonts.glob.scss importiert. Die wurde ja auch gelöscht

Reply:
Ja, aber welchen Nutzen hatte diese Datei?